validators/src/unit-test/java/uk/gov/moj/sdt/validators/BulkSubmissionValidatorTest.java

```diff
@@ -190,42 +195,38 @@ public void setUpLocalTests() {
      * create a bulk submission with one request, one customer, one individual requests and using the MCOL app
      */
     @Test
-    public void testAllSuccess() {
+    void testAllSuccess() {
 
         // set up a bulk customer to use the MCOL application
         bulkCustomer = createCustomer(createBulkCustomerApplications("MCOL"));
 
         // create an individual request
         final IIndividualRequest individualRequest = new IndividualRequest();
-        individualRequest.setId(requestId);
+        individualRequest.setId(REQUEST_ID);
         individualRequests = new ArrayList<>();
         individualRequests.add(individualRequest);
 
-        createBulkSubmission(numberOfRequests, bulkCustomer, individualRequests, "MCOL");
+        createBulkSubmission(NUMBER_OF_REQUESTS, bulkCustomer, individualRequests, "MCOL");
 
         // set up the mock objects
-        expect(mockIBulkCustomerDao.getBulkCustomerBySdtId(12345L)).andReturn(bulkCustomer);
-        replay(mockIBulkCustomerDao);
+        when(mockIBulkCustomerDao.getBulkCustomerBySdtId(12345L)).thenReturn(bulkCustomer);
```

Comment:
The value 12345L appears in many places in this class.  Probably best to replace it with a constant.